src/main/java/de/samply/share/client/model/centralsearch/UploadStats.java

```diff
@@ -1,15 +1,16 @@
 package de.samply.share.client.model.centralsearch;
 
-import org.simpleframework.xml.Element;
-import org.simpleframework.xml.Root;
+import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlProperty;
+import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlRootElement;
 
 /**
  * Represents the upload stats as received from central search.
  */
-@Root(name = "UploadStats")
+
+@JacksonXmlRootElement(localName = "UploadStats")
 public class UploadStats {
 
-  @Element(name = "LastUploadTimestamp")
+  @JacksonXmlProperty(localName = "LastUploadTimestamp")
   private String lastUploadTimestamp;
 
   public UploadStats() {
```

Comment:
Default constructor can be removed.

Reply:
Without constructor, xmlMapper cannot deserialize the string and throws an exception.

Reply:
Ok sorry, of corse. But the super() call can be removed, because that will be done implicitly.